README.md

```diff
@@ -81,9 +81,26 @@ See the [contributing page](http://jsonnet.org/contributing.html) on our website
 
 ### Running tests
 
-TODO
+```
+make && make libjsonnet.so &&  ./tests.sh
```

Comment:
You can just do `make test` to get this (and it also builds the little C programs that are tested).